src/incoming-message-handler/src/handlers/storeMessage.ts

```diff
@@ -1,29 +1,42 @@
 import type { AuditLog, S3PutObjectEvent } from "shared-types"
-import { AuditLogApiClient, AwsS3Gateway, createS3Config } from "shared"
+import { AuditLogApiClient, AwsS3Gateway, createS3Config, logger } from "shared"
 import { isError } from "shared-types"
 import readMessage from "../use-cases/readMessage"
 import { getApiKey, getApiUrl } from "../configs"
-import type { ValidationResult } from "../use-cases/retrieveMessageFromS3"
 import retrieveMessageFromS3 from "../use-cases/retrieveMessageFromS3"
 import formatMessage from "../use-cases/formatMessage"
+import CreateAuditLogUseCase from "src/use-cases/CreateAuditLogUseCase"
 
 interface StoreMessageResult {
   auditLog: AuditLog
   messageXml: string
 }
 
+export interface ValidationResult {
+  isValid: boolean
+  message?: string
+}
+
+interface StoreMessageValidationResult {
+  validationResult: ValidationResult
+}
+
 const s3Gateway = new AwsS3Gateway(createS3Config())
 const apiClient = new AuditLogApiClient(getApiUrl(), getApiKey())
+const createAuditLogUseCase = new CreateAuditLogUseCase(apiClient)
 
-export default async function storeMessage(event: S3PutObjectEvent): Promise<StoreMessageResult | ValidationResult> {
+export default async function storeMessage(
+  event: S3PutObjectEvent
+): Promise<StoreMessageResult | StoreMessageValidationResult> {
   const receivedMessage = await retrieveMessageFromS3(event, s3Gateway)
 
   if (isError(receivedMessage)) {
     throw receivedMessage
   }
 
-  if ("validationResult" in receivedMessage) {
-    return receivedMessage
+  if ("isValid" in receivedMessage) {
+    logger.info(JSON.stringify(receivedMessage))
```

Comment:
Nit: Should this be `info` or `debug`?

Reply:
This log here is not for debugging purposes. We are logging to cloud watch here to be able to use this info in monitoring and alerting later.